readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://bojkovladislav.github.io/layout_stars/)
+- [TEST REPORT LINK](https://bojkovladislav.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the discription
https://www.loom.com/share/da672f3d083c4247960d02d43f7d527a